src/azure-cli/azure/cli/command_modules/network/tests/latest/test_dns_commands.py

```diff
@@ -59,6 +59,14 @@ def _test_zone(self, zone_name, filename):
         # verify that each record in the original import is unchanged after export/re-import
         self._check_records(records1, records2)
 
+    @live_only()
```

Comment:
Just FYI, why decorated as `live_only`?

Because the HTTP request is never sent and vcr framework won't be able to record anything. However, vcr would think the test without YAML recording should run in live mode which requires login. 

So there is a conflict that CI won't perform and don't contain any login credential. This test is meant to be live only